content/it/security-chaos-engineering.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Security Chaos Engineering
+status: Completed
+category: Concetto
+tags: ["sicurezza", "metodologia", ""]
+---
+
+## Cos'è
+
+Security Chaos Engineering o SCE è una disciplina basata su [Chaos Engineering](/it/chaos-engineering/).
+SCE si avvale di sperimentazione proattiva della sicurezza su un sistema distribuito per assicurarsi che questo sia in grado di resistere a condizioni turbolente e dannose.
+Gli ingegneri del caos della sicurezza utilizzano cicli di metodi scientifici per raggiungere questo obiettivo,
+tra cui stato stazionario, ipotesi, verifica continua, lessons learned e implementazione delle mitigazioni.
+
+
+## Quali problematiche affronta
+
+La priorità principale per i [site reliability engineers](/it/site-reliability-engineering/) (SRE) e gli ingegneri della sicurezza informatica è
+ripristinare il servizio il più velocemente possibile con l'obiettivo di ottenere zero tempi di inattività e ridurre al minimo l'impatto sul business.
+Gli SRE e gli ingegneri della sicurezza informatica intervengono sia in situazioni critiche che precedono il guasto, sia post-mortem.
```

Comment:
Penso che con incident situations si voglia indicare le situazioni in cui l'incidente si è proprio verificato, dunque non solo una situazione critica. E il post-mortem è una valutazione a bocce ferme, mentre magari qui si sta prendendo in considerazione un tempo più prossimo all'incident.... 
```suggestion
Gli SRE e gli ingegneri della sicurezza informatica sono coinvolti in caso di incidente, sia prima, sia dopo il guasto.
```